google/cloud/spanner/value.h

```diff
@@ -311,27 +332,27 @@ class Value {
    *
    * This is mostly useful if writing generic code where casting is needed.
    *
-   * Example:
+   * @par Example
    *
-   *     spanner::Value v{true};
-   *     bool b = static_cast<bool>(v);  // OK: b == true
+   * @code
+   * spanner::Value v{true};
+   * bool b = static_cast<bool>(v);  // OK: b == true
    *
-   *     spanner::Value null_v = spanner::MakeNullValue<std::int64_t>();
-   *     std::int64_t i = static_cast<std::int64_t>(null_v);  // UB! null
-   *     bool bad = static_cast<bool>(null_v);  // UB! wrong type
+   * spanner::Value null_v = spanner::MakeNullValue<std::int64_t>();
+   * std::int64_t i = static_cast<std::int64_t>(null_v);  // UB! null
+   * bool bad = static_cast<bool>(null_v);  // UB! wrong type
+   * @endcode
    */
   template <typename T>
   explicit operator T() const {
     return *get<T>();
   }
 
-  /**
-   * Allows Google Test to print internal debugging information when test
-   * assertions fail.
-   *
-   * @warning This is intended for debugging and human consumption only, not
-   * machine consumption as the output format may change without notice.
-   */
+  // Allows Google Test to print internal debugging information when test
```

Comment:
This one no longer has Doxygen comments, is that intentional?  I think it should have them, as the function is part of the public API (sadly that is what gtest requires).

Reply:
I did remove the doxygen comment on purpose, but I've added it back.